Core/Src/main.c

```diff
@@ -149,6 +151,7 @@ int main(void)
   /* USER CODE BEGIN RTOS_THREADS */
   temp_monitor_handle = osThreadNew(vTempMonitor, &hi2c1, &temp_monitor_attributes);
   watchdog_monitor_handle = osThreadNew(vWatchdogMonitor, GPIOB, &watchdog_monitor_attributes);
+  imu_monitor_handle = osThreadNew(vIMUMonitor, &hi2c2, &imu_monitor_attributes);
```

Comment:
I think this is still `hi2c1.` Ima make a ticket for mutexing the I2C bus if I didn't already